news/2018-10-13-osu-world-cup-2018-registrations-open.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "osu! World Cup 2018: Registrations now open!"
+date: 2018-10-13 13:00:00 +0000
+---
+
+Circle it on your calendars, because OWC 2018 registrations are now open!
+
+![](/wiki/shared/news/banners/OWC_2018_banner.png)
+
+For more details, refer to the **[osu! World Cup 2018 wiki article](/wiki/Tournaments/OWC/2018/)**. You can discuss the event and follow its most important changes in the **[official discussion thread](https://osu.ppy.sh/community/forums/topics/815745)**.
+ 
+## Tournament Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | Oct 13th - Oct 26th |
+| Live Drawings | Nov 3rd  (14:00 UTC) |
+| Group Stage | Nov 10th & 11th |
+| Round of 16 | Nov 17th & 18th |
+| Quarterfinals | Nov 24th & 25th |
+| Semifinals | Dec 1st & 2nd |
+| Finals | Dec 8th & 9th |
+| Grand Finals | Dec 15th & 16th |
+ 
+## Prizes
+
+- **1st:** US$300 per team member, exclusive single-run merch, profile badge, "osu! Champion" user title for one year  
+- **2nd:** US$160 per team member, exclusive single-run merch, Profile badge  
+- **3rd:** US$80 per team member, exclusive single-run merch, Profile badge  
+ 
+## How do I register?
+ 
+- **[Click here to sign-up!](https://osu.ppy.sh/community/tournaments/17)**
+- Do not create teams in advance! Teams will be formed by the country's team captain after the registration phase is concluded.
+- Hand in your registration before the **27th of October!**
```

Comment:
should be `27th October`

Reply:
Should it? Registration phase finishes the 26th at 23:59 UTC so, makes sense it says 27 right?